src/System.Private.ServiceModel/tests/Scenarios/Contract/Message/MessageContractCommon.4.1.0.cs

```diff
@@ -40,6 +40,12 @@ public static MessageHeaders GetHeaders(MyInspector inspector)
             return headers;
         }
 
+        public static MessageProperties GetProperties(MyInspector inspector)
```

Comment:
>public static MessageProperties GetProperties(MyInspector inspector) [](start = 8, length = 68)

This change is not actually needed for this PR and will be removed before merging.